ddp-workspace/projects/ddp-sdk/src/lib/services/activity/activityValidatorBuilder.service.ts

```diff
@@ -52,6 +52,7 @@ export class ActivityValidatorBuilder {
             { type: ValidationRuleType.AgeRange, factory: (x, y) => this.buildAgeRangeValidator(x, y) },
             { type: ValidationRuleType.IntRange, factory: (x, y) => this.buildNumericRangeValidator(x, y) },
             { type: ValidationRuleType.Unique, factory: (x, y) => new ActivityUniqueValidationRule(y) },
+            { type: ValidationRuleType.UniqueValue },
```

Comment:
In order to prevent a logger error "**_Received unknown type of validation rule.._**" (see [lines 73-74](https://github.com/broadinstitute/ddp-angular/pull/1084/files#:~:text=this.logger.logError(this,rule%20named%3A%20%24%7BvalidationJson.rule%7D%60)%3B))

Reply:
Since we have more server-side validations defined in a question definition (e.g. **unique_value** /  **cross-form** validations)
there is a question: 
  Should we use such approach for those validations 
    or  
  we can just write the error (mentioned above) in console (but in Stackdriver errors as well).

What do you think, @MocanaAtBroad ?